### PR TITLE
KAFKA-3709: Create project security page

### PR DESCRIPTION
**Pull request by fpj**

### CODE REVIEW

File: docs/apache_kafka_project_security.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+-->
+
+<!--#include virtual="../includes/header.html" -->
+
+<h1>Kafka Security</h1>
+
+<p>
+	The Apache Software Foundation takes security issues very seriously. Apache Kafka specifically offers security features and is responsive to issues around its features. If you have any concern around Kafka Security or believe you have uncovered a vulnerability, we suggest that you get in touch via the e-mail address <a href="mailto:security@kafka.apache.org?Subject=[SECURITY] My security issue" target="_top">security@kafka.apache.org</a>. In the message, try to provide a description of the issue and ideally a way of reproducing it. The security team will get back to you after assessing the description.
+
+	Note that this security address should be used only for undisclosed vulnerabilities. Dealing with fixed issues should be handled regularly via the user and the dev lists. <b>Please report any security problems to the project security address before disclosing it publicly.</b>
```
Comment (ijuma) [inline docs/apache_kafka_project_security.html]: Maybe we should complement `Dealing with fixed issues or general user questions...`. What do you think?

**Comment by ijuma** [comment]
Flavio, thanks for the PR! I think it probably needs to be done in the website repo: https://github.com/apache/kafka-site

The `docs` folder is only for things that go under `documentation` on the website, I imagine that this would be a sibling, right?

We also need to add a link to this page in the left-hand navbar.

The page itself looks great, I just had one minor comment.

File: docs/apache_kafka_project_security.html (hunk 1 above)
Comment (fpj) [inline docs/apache_kafka_project_security.html]: Good suggestion, check if this is better.

**Comment by fpj** [comment]
@ijuma Ok, will create a PR agains the project site repo.

File: docs/apache_kafka_project_security.html (hunk 1 above)
Comment (ijuma) [inline docs/apache_kafka_project_security.html]: Looks good, thanks.

**Pull request closed without merging**